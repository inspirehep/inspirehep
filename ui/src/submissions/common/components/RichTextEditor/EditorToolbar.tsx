import React, { Component } from 'react';

class EditorToolbar extends Component {
  render() {
    return (
      <>
        <span className="ql-formats">
          <button className="ql-bold" type="button" title="Bold" />
          <button className="ql-italic" type="button" title="Italic" />
        </span>
        <span className="ql-formats">
          <button
            className="ql-list"
            value="ordered"
            type="button"
            title="Numbered List"
          />
          <button
            className="ql-list"
            value="bullet"
            type="button"
            title="Bulleted List"
          />
        </span>
        <span className="ql-formats">
          <button className="ql-link" type="button" title="Link" />
        </span>
        <span className="ql-formats">
          <button
            className="ql-clean"
            type="button"
            title="Clear All Formatting"
          />
        </span>
      </>
    );
  }
}

export default EditorToolbar;
