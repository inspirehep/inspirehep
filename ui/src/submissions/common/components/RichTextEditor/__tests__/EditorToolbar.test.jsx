import React from 'react';
import { render } from '@testing-library/react';

import EditorToolbar from '../EditorToolbar';

describe('RichTextEditor', () => {
  it('renders', () => {
    const { asFragment } = render(<EditorToolbar />);

    expect(asFragment()).toMatchSnapshot();
  });
});
