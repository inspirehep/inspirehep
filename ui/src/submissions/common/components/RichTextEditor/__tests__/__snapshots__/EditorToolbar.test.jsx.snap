// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RichTextEditor renders 1`] = `
<DocumentFragment>
  <span
    class="ql-formats"
  >
    <button
      class="ql-bold"
      title="Bold"
      type="button"
    />
    <button
      class="ql-italic"
      title="Italic"
      type="button"
    />
  </span>
  <span
    class="ql-formats"
  >
    <button
      class="ql-list"
      title="Numbered List"
      type="button"
      value="ordered"
    />
    <button
      class="ql-list"
      title="Bulleted List"
      type="button"
      value="bullet"
    />
  </span>
  <span
    class="ql-formats"
  >
    <button
      class="ql-link"
      title="Link"
      type="button"
    />
  </span>
  <span
    class="ql-formats"
  >
    <button
      class="ql-clean"
      title="Clear All Formatting"
      type="button"
    />
  </span>
</DocumentFragment>
`;
